app/views/public/_eventstatus.html.erb

```diff
@@ -0,0 +1,11 @@
+<% @user = current_user %>
+<% @event = event %>
+<% if @user &.event? @event %>
+    <%= link_to 'Delete', event_path(@event), method: :delete, data: {confirm: "Are you sure you want to delete this event?"}, class: "action-btn" %>
+    <% end %>
+<% if @user &.attended_event? @event %>
+    <%= link_to 'Leave',  attendaces_path(event_id: @event.id), method: :delete, data: {confirm: "Leave this event?" }, class: "action-btn" %>
+<% end %>
+<% if @event.upcoming? %>
+    <%= link_to 'Attend', attendaces_path(event_id: @event.id), method: :post, data: {confirm: "Attend this event?" }, class: "action-btn" %>
```

Comment:
I get this error when I try to attend an event: #3.

Reply:
hi, @efrapp I think that particular problem in attending an event from users/show. is from your end. probably not running "bundle update before starting the application" it works fine on my machine, I would update the code